graphql/image_resolver.go

```diff
@@ -27,6 +30,26 @@ func (r *imageResolver) Distros(ctx context.Context, obj *thirdparty.Image) ([]*
 	return apiDistros, nil
 }
 
+// Packages is the resolver for the packages field.
+func (r *imageResolver) Packages(ctx context.Context, obj *model.APIImage, opts thirdparty.PackageFilterOptions) ([]*model.APIPackage, error) {
+	config, err := evergreen.GetConfig(ctx)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("getting evergreen configuration: '%s'", err.Error()))
+	}
+	c := thirdparty.NewRuntimeEnvironmentsClient(config.RuntimeEnvironments.BaseURL, config.RuntimeEnvironments.APIKey)
+	packages, err := c.GetPackages(ctx, opts)
```

Comment:
I don't think this works in practice because opts is missing the AMI field. I would assume it fails on `localhost:9090/graphql`

Reply:
eek - makes sense - fixed!